src/central-server/admin-service/ui-system-test/src/intTest/java/org/niis/xroad/cs/test/ui/glue/TrustServicesOcspRespondersStepDefs.java

```diff
@@ -48,8 +48,21 @@ public void ocspRespondersTabIsSelected() {
         trustServicesPageObj.certServiceDetails.tabOcspResponders().scrollIntoView(false).click();
     }
 
-    @Step("OCSP responder with URL {} is added")
-    public void newOcspResponderIsAdded(String url) throws Exception {
+    @Step("OCSP responder with URL {string} is added")
+    public void newOcspResponderIsAdded(String url) {
+        ocspRespondersPageObj.btnAddOcspResponder().click();
+        commonPageObj.dialog.btnCancel().should(Condition.enabled);
```

Comment:
`enabled`could also be static imported like `visible` and some other Conditions in this file